tests/network/sriov.go

```diff
@@ -695,3 +670,28 @@ func findIfaceByMAC(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineIns
 	}
 	return ifaceName, nil
 }
+
+func newSRIOVVmi(networks []string, cloudInitNetworkData string) *v1.VirtualMachineInstance {
+	options := []libvmi.Option{
+		libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+		libvmi.WithNetwork(v1.DefaultPodNetwork()),
+	}
+	if cloudInitNetworkData != "" {
+		cloudinitOption := libvmi.WithCloudInitNoCloudNetworkData(cloudInitNetworkData)
+		options = append(options, cloudinitOption)
+	}
+
+	for _, name := range networks {
+		options = append(options,
+			libvmi.WithInterface(libvmi.InterfaceDeviceWithSRIOVBinding(name)),
+			libvmi.WithNetwork(libvmi.MultusNetwork(name, name)),
```

Comment:
a bit of an off topic, so I'll leave it as suggestion only: using same name for NAD and network name does not allow us to run tests with 2 NICS connected to same NAD, is that something you want to address in this PR as well?

Reply:
This scenario has not existed so far and it was not there even in the previous helper.
I think we should adjust it accordingly when the need lands.
At that time, we may need a further refactoring done here (e.g. split between single secondary network and multi secondary networks).

Reply:
yeah totally, just wanted to mention that it's something we may want to tackle in the future..